Qpf/Macro/Data/Replace.lean

```diff
@@ -19,10 +19,11 @@ structure CtorArgs where
 
 /- TODO(@William): make these correspond by combining expr and vars into a product -/
 structure Replace where
-  (expr: Array Term)
-  (vars: Array Name)
+  (vals: Array (Name × Term))
```

Comment:
`vals` and `vars` is much too similar for my taste. i'd suggest `newParameters`.
We can be a bit larger, since we'll use the shorter projections usually anyways
```suggestion
  (newParameters : Array (Name × Term))
```